deploy/helm/flightctl/charts/keycloak/values.yaml

```diff
@@ -34,6 +34,7 @@ realm:
   ## @param redirectUris The redirect URIs for Flight Control
   redirectUris:
     - /realms/flightctl/account/*
+    - http://127.0.0.1/callback
```

Comment:
When using the `flightctl login --web` command, it will start a local http server https://github.com/flightctl/flightctl/blob/main/internal/cli/login.go#L153 .This is needed to obtain the authorization code in Oauth2 Standard flow via front channel which is then exchanged for the access code via back channel.

I dont think this is something that should to be configurable as it needs to align with the CLI code.